en/user/articles-dxp/120-search/13-result-rankings.markdown

```diff
@@ -0,0 +1,184 @@
+
+1.  Designate that certain results should appear at the top of the results if
+    they are matched with a certain keyword. This is the idea of _pinning_
+    results to the top of the results list.
+
+2.  By contrast, hide results that shouldn't appear in certain searches at all. 
+
+3.  Add results that aren't normally returned by searching a certain keyword.
+
+4.  Re-order results with a drag-and-drop interface.
+
+<!-- Not sure if we need to show this 5.  Configure a search page to see the results from the Result Rankings
+    customization and the un-manipulated results. LPS-96212
+-->
+
+Result Rankings lets you pin, hide, and add search results for a given set of
+keywords.
+
+| **Use Case:** At the Lunar Resort website, interested site visitors often search
+| for specific activities, entering keywords like "rover races", "atv rentals",
+| and "lunar golf". For all of these, the Lunar Resort wants a certain [Content
+| Page](/docs/7-2/user/-/knowledge_base/u/creating-content-pages) to always appear
+| at the top of the search results. This is the go-to Activities page in the Lunar
+| Resort where guests can go to find all of the resort's adventurous offerings,
+| including lunar rover races, ATV rentals, and information about golfing
+| packages. By contrast, the Lunar  Resort does not want the legal liability
+| waiver form to appear during a search for fun activities: that's a bridge to be
+| crossed once guests are already signing up for the activity. It shouldn't
+| pollute a search for fun activities, even though it contains many of the
+| keywords Users would search for. Result Rankings lets you, the portal
+| administrator, _pin_ the Activities Content Page to the top of the results and
+| _hide_ the liability waiver Web Content Article.  In addition, a community
+| member wrote a blog favorably reviewing the Lunar Resort, and you want that
+| content added to searches for activities at the resort. This is a prime use case
+| for Result Rankings. 
+
+![Figure 1: The Lunar Resort wants to tweak these results: pin the Activities page to the top, and hide the legal content entirely.](../../images/search-result-rankings-todo.png) 
+
+## Availability
+
+Search Tuning features like Result Rankings are only supported when using
+Elasticsearch as the search engine.
+
+Results Rankings was added in @product-ver@ Service Pack 1.
+
+## Requirements and Limitations
+
+Search Tuning features like Result Rankings are only supported when using
+Elasticsearch as the search engine.
+
+The Result Ranking entries are currently shared across all Virtual Instances so it is not possible to customize your search results differently. Because of this multi-tenant @product@ integration with Elasticsearch (connecting multiple, indepedent @procuct@ deployments to the same Elasticsearch cluster) is not supported either currently when Result Rankings is also used. [LPS-101291](https://issues.liferay.com/browse/LPS-101291)
```

Comment:
I want to clarify this but, I don't want to make a false statement. This is my understanding:

> Result Ranking entries are currently configurable at the Virtual Instance scope, so it is not possible to customize your search results per site. In addition, Result Rankings created in a Virtual Instance are not scoped only to that Virtual Instance, but will be shared across all @product@ Virtual Instances in the deployment, and even across separate deployments in a multi-tenant Elasticsearch cluster. Therefore, Result rankings shouldn't be used when connecting multiple @product@ deployments to the same Elasticsearch cluster unless you intend for the same Result rankings to apply to every @product@ deployment. See [LPS-101291](https://issues.liferay.com/browse/LPS-101291) for more information.

 Is this accurate?

Reply:
It's accurate though it sounds confusing at first reading: we start with 
>Result Ranking entries are currently configurable at the Virtual Instance scope,

but then we say 
> In addition, Result Rankings created in a Virtual Instance are not scoped only to that Virtual Instance, but will be shared across all @Product@ Virtual Instances in the deployment, 

What if we say something like
>Result Ranking entries are not scoped to a Virtual Instance, but will be shared across all @Product@ Virtual Instances in the deployment, and even across separate deployments if they share an Elasticsearch cluster (in a multi-tenant scenario).
>Therefore, Result rankings shouldn't be used when connecting multiple @Product@ deployments to the same Elasticsearch cluster unless you intend for the same Result rankings to apply to every @Product@ deployment. See LPS-101291 for more information.

What do you think about it?

Reply:
Pretty good. I only changed it a little bit:

> Result Ranking entries are configured in a Virtual Instance, but are not applied only to that Virtual Instance. Instead, custom rankings made in one virtual Instance are shared across all @product@ Virtual Instances in the deployment, and even across separate deployments sharing an Elasticsearch cluster (in a multi-tenant scenario).  Therefore, Result rankings shouldn't be used when connecting multiple @product@ deployments to the same Elasticsearch cluster unless you intend for the same Result rankings to apply to every @product@ deployment. See [LPS-101291](https://issues.liferay.com/browse/LPS-101291) for more information.